src/components/TransactionalInfo.tsx

```diff
@@ -16,35 +16,185 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
+import { calAmountDelta } from '@alephium/sdk'
+import { Output, Transaction } from '@alephium/sdk/api/explorer'
+import { useTranslation } from 'react-i18next'
+import { useParams } from 'react-router-dom'
 import styled, { useTheme } from 'styled-components'
 
+import { AddressHash, SimpleTx, useAddressesContext } from '../contexts/addresses'
+import { isExplorerTransaction, isSimpleTx, TransactionDirection } from '../utils/transactions'
+import AddressBadge from './AddressBadge'
 import Amount from './Amount'
+import Badge from './Badge'
+import DirectionalArrow from './DirectionalArrow'
+import IOList from './IOList'
+import TimeSince from './TimeSince'
+import Token from './Token'
 
 interface TransactionalInfoProps {
-  type: 'out' | 'in' | 'pending'
-  content: string | bigint
+  transaction: Transaction | SimpleTx
```

Comment:
How about renaming `SimpleTx` to `PendingTx`?

/cc @mvaivre

Reply:
Seems out of scope to do it in this PR but it was quick to do, so it's done